src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: article
+title: Keyboard
+description: Appwrite's approach to keyboard accessibility, how to use the keyboard, and how to use the Appwriter.
+---
+
+The Appwrite SDK console is optimized with the keyboard using shortcut keys to enable accessibility.  Using shortcut keys access websites without the mouse. 
```

Comment:
> using shortcut key access websites without the mouse.

This sounds like a fragment 🧐

Reply:
Surprisingly, it is not!

Reply:
See eldad's comments. 

Also, do any of our content/style guidelines say to use 2 spaces after a period? If not, let's stick with 1 as it will cut down on the number of characters and the extra space isn't rendered.